src/pkg/cli/compose/validation.go

```diff
@@ -274,14 +276,21 @@ func ValidateProject(project *compose.Project) error {
 
 		for k := range svccfg.Extensions {
 			switch k {
-			case "x-defang-dns-role", "x-defang-static-files", "x-defang-redis", "x-defang-postgres":
+			case "x-defang-redis", "x-defang-postgres":
+				numOfManagedServices++
+				continue
+			case "x-defang-dns-role", "x-defang-static-files":
 				continue
 			default:
 				term.Warnf("service %q: unsupported compose extension: %q", svccfg.Name, k)
 			}
 		}
 	}
 
+	if numOfManagedServices > 0 && numOfManagedServices == len(project.Services) {
```

Comment:
Let's allow this, but simply revert to the old CLI behavior, ie. dont' wait for deployment events